README.md

```diff
@@ -4,20 +4,11 @@ This project was bootstrapped with [Create Vite](https://github.com/vitejs/vite/
 
 ## Development
 
-### pnpm
-
-This project uses [pnpm](https://pnpm.io/) as a package manager. In many of our repositories we use a monorepo approach. The optimal solution for this approach is to use `pnpm`. It is faster than `npm` and consumes less disk storage than `yarn`.
```

Comment:
What's the reason to migrate from `pnpm`? It's stable, actively developed, much faster and pretty much innovate and efficient when it comes to memory management - node_modules has symbolic links to shared source of packages, local versioning of packages per case.

Reply:
I think this is because I hadn't committed to pnpm yet.

Consider this my commitment to `pnpm`, we can remove the commit that makes this switch and force-push it away.

Although y'all, if you haven't tried `bun`, that's some fast dependency resolution :grin: (We will not be switching to bun. Yet.)